frontend/src/app/components/wp-table/configuration-modal/wp-table-configuration-relation-selector.ts

```diff
@@ -35,21 +35,22 @@ export class WpTableConfigurationRelationSelectorComponent implements OnInit  {
   public selectedRelationFilter:QueryFilterResource|undefined = undefined;
 
   public text = {
+    filter_work_packages_by_relation_type: this.I18n.t('js.work_packages.table_configuration.relation_filters.filter_work_packages_by_relation_type'),
     please_select: this.I18n.t('js.placeholders.selection'),
-    first_part:    this.I18n.t('js.work_packages.table_configuration.relation_filters.first_part'),
-    second_part:   this.I18n.t('js.work_packages.table_configuration.relation_filters.second_part'),
-    parent:        this.I18n.t('js.types.attribute_groups.filter_types.parent'),
-    precedes:      this.I18n.t('js.types.attribute_groups.filter_types.precedes'),
-    follows:       this.I18n.t('js.types.attribute_groups.filter_types.follows'),
-    relates:     this.I18n.t('js.types.attribute_groups.filter_types.relates'),
-    duplicates:    this.I18n.t('js.types.attribute_groups.filter_types.duplicates'),
-    duplicated:  this.I18n.t('js.types.attribute_groups.filter_types.duplicated'),
-    blocks:        this.I18n.t('js.types.attribute_groups.filter_types.blocks'),
-    blocked:     this.I18n.t('js.types.attribute_groups.filter_types.blocked'),
-    requires:      this.I18n.t('js.types.attribute_groups.filter_types.requires'),
-    required:    this.I18n.t('js.types.attribute_groups.filter_types.required'),
-    partof:        this.I18n.t('js.types.attribute_groups.filter_types.partof'),
-    includes:      this.I18n.t('js.types.attribute_groups.filter_types.includes')
+    // We need to inverse the translation strings, as the filters's are named the other way around than what
+    // a user knows from the relations tab:
+    parent:        this.I18n.t('js.relation_labels.children'),
+    precedes:      this.I18n.t('js.relation_labels.follows'),
+    follows:       this.I18n.t('js.relation_labels.precedes'),
+    relates:     this.I18n.t('js.relation_labels.relates'),
+    duplicates:    this.I18n.t('js.relation_labels.duplicated'),
+    duplicated:  this.I18n.t('js.relation_labels.duplicates'),
+    blocks:        this.I18n.t('js.relation_labels.blocked'),
+    blocked:     this.I18n.t('js.relation_labels.blocks'),
+    requires:      this.I18n.t('js.relation_labels.required'),
+    required:    this.I18n.t('js.relation_labels.requires'),
+    partof:        this.I18n.t('js.relation_labels.includes'),
+    includes:      this.I18n.t('js.relation_labels.partof')
```

Comment:
This will invalidate all translations existing in languages translated on crowdin. I am by no means against changing the keys, but we should probably not be doing it this short before a release and if we do, we need to have the necessary translations added.

Reply:
The `relation_labels.*` translations all exist for quite some time, only the special "attribute_groups" have been removed

Reply:
You are right of course @oliverguenther. I mistakenly interpreted this to be the source (yml), so nevermind.